tests/test_translation.py

```diff
@@ -933,135 +926,136 @@ def test_translation_correct(
-                "to_unique_index": new_to_unq,
-            }
-            msg = "Duplicate values found in from_unique_index"
-        else:
-            raise ValueError("This shouldn't happen! Debug code.")
-
-        # Run with errors
-        with pytest.raises(ValueError, match=msg):
-            translation.pandas_single_vector_zone_translation(
-                **(pd_vector_split.input_kwargs() | new_kwargs)
-            )
-
-    def test_non_vector(self, pd_vector_split: PandasVectorResults):
-        """Test that an error is thrown when a non-vector is passed in"""
-        new_vector = pd.DataFrame(pd_vector_split.vector)
-        new_vector[1] = new_vector[0].copy()
-        with pytest.raises(ValueError, match="must be a pandas.Series"):
-            translation.pandas_single_vector_zone_translation(
-                **(pd_vector_split.input_kwargs() | {"vector": new_vector})
-            )
-
-    @pytest.mark.parametrize(
-        "pd_vector_str",
-        ["pd_vector_aggregation", "pd_vector_split"],
-    )
-    def test_series_like(self, pd_vector_str: str, request):
-        """Test vector-like arrays (empty in 2nd dim)"""
-        pd_vector = request.getfixturevalue(pd_vector_str)
-        new_vector = pd.DataFrame(pd_vector.vector)
-        result = translation.pandas_single_vector_zone_translation(
-            **(pd_vector.input_kwargs() | {"vector": new_vector})
-        )
-        pd.testing.assert_series_equal(result, pd_vector.expected_result, check_names=False)
-
-    @pytest.mark.parametrize(
-        "pd_vector_str",
-        ["pd_vector_aggregation", "pd_vector_split"],
-    )
-    @pytest.mark.parametrize("check_totals", [True, False])
-    def test_translation_correct(
-        self,
-        pd_vector_str: str,
-        check_totals: bool,
-        request,
-    ):
-        """Test that aggregation and splitting give correct results
-
-        Also checks that totals are correctly checked.
-        """
-        pd_vector = request.getfixturevalue(pd_vector_str)
-        result = translation.pandas_single_vector_zone_translation(
-            **pd_vector.input_kwargs(check_totals=check_totals)
-        )
-        pd.testing.assert_series_equal(result, pd_vector.expected_result)
-
-    @pytest.mark.parametrize(
-        "pd_vector_str",
-        ["pd_vector_aggregation", "pd_vector_split"],
-    )
-    def test_check_allow_similar_types(
-        self,
-        pd_vector_str: str,
-        request,
-    ):
-        """Test that similar types are allowed in translation and data."""
-        pd_vector = request.getfixturevalue(pd_vector_str)
-        new_trans = pd_vector.translation.copy()
-        new_trans.from_col = new_trans.from_col.astype(np.int32)
-        result = translation.pandas_single_vector_zone_translation(
-            **(pd_vector.input_kwargs() | {"translation": new_trans.df})
-        )
-        pd.testing.assert_series_equal(result, pd_vector.expected_result)
+# class TestPandasVector:
```

Comment:
I think it is worth keeping a single test for the single vector translation even if its just a duplicate of one of the multi vector tests just incase the function gets edited in future.

As for the other tests in here can they just be removed instead of commented out.